src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Common/Capabilities.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT license. See License.txt in the project root for license information.
+
+using Microsoft.AspNetCore.Razor.LanguageServer.Semantic;
+using Microsoft.VisualStudio.LanguageServer.Protocol;
+
+namespace Microsoft.AspNetCore.Razor.LanguageServer.Common;
+
+internal static class Capabilities
+{
+    public static void DocumentColorProvider(VSInternalServerCapabilities serverCapabilities)
```

Comment:
Should these methods be extension methods on `VSInternalServerCapabilities`? E.g. `serverCapabilities.EnableDocumentColorProvider()`?

Reply:
@davidwengier thoughts? I like this idea and the `Enable` wording

Reply:
Yep, an extension method is a great idea, and `Enable` is just as good a verb as `Apply` :D